tst/EnergyPlus/unit/DXCoils.unit.cc

```diff
@@ -3557,8 +3557,8 @@ TEST_F(EnergyPlusFixture, TestMultiSpeedCoilsAutoSizingOutput)
     EXPECT_EQ(0.875, DXCoil(1).MSRatedAirVolFlowRate(2) * 0.5);
     EXPECT_EQ(0.875, DXCoil(1).MSRatedAirVolFlowRate(1));
     // Design Capacity at speed 2 and speed 1
-    EXPECT_EQ(31899.037804634620, DXCoil(1).MSRatedTotCap(2));
-    EXPECT_EQ(31899.037804634620 * 0.5, DXCoil(1).MSRatedTotCap(1));
+    EXPECT_NEAR(31888, DXCoil(1).MSRatedTotCap(2), 0.01);
```

Comment:
Probably make the expected result ```31888.0```. Not sure if this would ever be interpreted as an ```int``` as it is currently.